api/module.mk

```diff
@@ -33,14 +33,14 @@ JAR_LINK     := $(DERIVED_TARGET_DIR)/wfl.jar
 
 $(PREBUILD): $(SCM_SRC) $(SCM_RESOURCES)
```

Comment:
I think you can remove the dependence on SCM_SRC here as the only thing that affects prebuild is your new build.clj file + this + deps.edn?

Reply:
Yes. Thanks.